ETR01SDK-630: Docs: refactor CMake options mentions in tutorials (#537)
tropicsquare/libtropic: docs/tutorials/stm32/identify_chip.md

    After this, you should see a colored output in your serial monitor.

## Configuration
Besides the [Libtropic CMake options](../../reference/integrating_libtropic/how_to_configure/index.md) used to configure Libtropic, there is the following CMake option specific to this example:

> - `STLINK_SERIAL_NUMBER` (default: none, OpenOCD looks for any STLink programming interface) to set the serial number of the STLink device which will be used for flashing. This is needed only when you have multiple STM32s connected using built-in STLink or problems with OpenOCD's autodetection:

Reviewer: Same as the previous comment. This is one of the suggestions, feel free to modify/change entirely.

```suggestion
STLINK_SERIAL_NUMBER (default: none, OpenOCD looks for any STLink programming interface) specifies the serial number of the STLink device used for flashing. This is needed only when you have multiple STM32s connected via built-in STLink or when OpenOCD autodetection does not work:
```
